app/src/main/java/combi/example/andre/finalproject/Homepage.java

```diff
@@ -6,8 +6,12 @@
 import android.support.design.widget.Snackbar;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.Toolbar;
+import android.util.Log;

```

Comment:
Not used anymore, might want to remove?

Reply:
Yeah, the changes here only add import, which is weird. This can be cleaned up, and android studio can do the cleanup for you 


![2018-06-03 17_35_23-myapplication3 c__users_seiry_documents_github_myapplication3 - _app_src_ma](https://user-images.githubusercontent.com/1240672/40885633-8f9bb588-6754-11e8-93f7-78bbc435254e.png)